eng/Version.Details.xml

```diff
@@ -148,39 +148,39 @@
       <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
       <Sha>9d5a6a9aa463d6d10b0b0ba6d5982cc82f363dc3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.App.Ref" Version="9.0.1">
+    <Dependency Name="Microsoft.AspNetCore.App.Ref" Version="9.0.0">
```

Comment:
Out of curiosity, how did you change these versions? If you did it through darc, I would have expected the Sha's to change.

Reply:
I change those manually. Darc pulls the non-public versions (i.e., 9.0.1)